### DISCUSSION TITLE
How to fix error lua:67 and lua81

### DISCUSSION THREAD

**Discussion by Aquiverll** [community/community#27848] [Codespaces]
first of all my english is not very good
I keep getting errors on CHEAT TABLE. 
I tried every way, even formatted my computer.but ı cant solved PLS HELP  @xAranaktu
![resim_2022-08-01_072943076](https://user-images.githubusercontent.com/109752303/182072457-f4783189-37db-4be5-a185-610b2b5bcbe2.png)

**Comment by MrSarno** [comment]


**Reply by Aquiverll** [reply]
You wrote that I won't help because the game is an online game, but I'm playing FIFA 19. I was just trying to make a few changes in career mode.

**Comment by Aquiverll** [comment]
![Ekran görüntüsü 2022-08-01 074305](https://user-images.githubusercontent.com/109752303/182073926-344d463d-6c7c-496b-a14d-c46bafed8f92.png)
![Ekran görüntüsü 2022-08-01 074321](https://user-images.githubusercontent.com/109752303/182073937-386e6f1e-7e8c-4919-bc44-d59cf41f0659.png)

**Comment by xAranaktu** [comment]
This happens probably because the cheat table is outdated. The source from where the table is loading images is no longer available and therefore doesn't work. There isn't any other fix than editing the source code, removing all the image requests.

**Reply by Aquiverll** [reply]
thanks for answer
